.gitignore

```diff
@@ -76,3 +76,7 @@ yarn-error.log
 src/data/contributors.json
 # These files are generated by `yarn merge-translations` command
 src/intl/*.json
+
+# IDE
+.idea/
+.viscode/
```

Comment:
`.vscode/`

Reply:
Hey, my logic is that: all bugs are patches, not all patches are bugs. https://semver.org/
Reworking the bug template would be a better solution imo.
I'd say do what works best for the team.
Best regards

Reply:
(@timhc22 note line 82 should be `.vscode/`)
(@samajammin bump 👆 any thoughts on this issue type?)

Reply:
🧐 ah yes! should be updated now